autoendpoint/src/extractors/subscription.rs

```diff
@@ -265,10 +265,14 @@ fn validate_vapid_jwt(
     let VapidHeaderWithKey { vapid, public_key } = vapid;
 
     let public_key = decode_public_key(public_key)?;
+    let mut validation = Validation::new(Algorithm::ES256);
+    validation.set_audience(&["https://push.services.mozilla.org", "http://127.0.0.1:9160"]);
```

Comment:
I think these should be passed in as configuration options. I really prefer if we don't hard code things (particularly the test URL.)

Reply:
Updated the `Settings` struct to hold the vapid_aud value as a `vec`, then pull it in to pass to the `set_audience` method. If you prefer another method with env vars and whatnot, can do that. Do we use the [`cloudops-infra` repo for `autopush`](https://github.com/mozilla-services/cloudops-infra/tree/master/projects/autopush) for env var storage and reading, btw?

Reply:
No, that should be fine. I've tested locally and things are working fine either if it's specified in a config file 

```toml
vapid_aud = ["https://push.services.mozilla.org", "http://127.0.0.1:8001"]
```

or as an environment variable
```bash
AUTOENDPOINT_VAPID_AUD='["https://push.services.mozilla.org", "http://127.0.0.1:8001"]'
```
(Older versions did not handle parsing the environment string correctly. Looks like that's no longer a problem.)